SharedPluginFiles/FormItInterface.mod.js

```diff
@@ -0,0 +1,186 @@
+const FormItInterface = {};
```

Comment:
This file is supposed to be the same as FormItInterface.js, but exports a module so that it can be used in import statements. Would probably be better if this is in typescript and then the appropriate files are generated with something like webpack and published on npm.